### PR TITLE
Added FiveM and Node js support

### PR DESCRIPTION
**Pull request by zigazajc007**

### CODE REVIEW

File: nodejs/nodejs.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "type": "nodejs",
+  "display": "Node JS",
```
Comment (Romitou) [inline nodejs/nodejs.json:3]:
```suggestion
  "display": "NodeJS",
```

File: nodejs/nodejs.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "run": {
+    "stop": "stop",
```
Comment (Romitou) [inline nodejs/nodejs.json:5]: Is this correct? In my opinion, a signal 2 (SIGINT) should be sent.

File: nodejs/nodejs.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      "type": "standard"
+    }
+  ],
+  "data": {
+    "memory": {
+      "value": "1024",
+      "type": "integer",
+      "required": true,
+      "desc": "How much memory in MB to allocate to the Node js",
```
Comment (Romitou) [inline nodejs/nodejs.json:27]:
```suggestion
      "desc": "How much memory in MB to allocate to the NodeJS process",
```

File: nodejs/nodejs.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      "display": "Memory (MB)",
+      "internal": false
+    },
+    "start": {
+      "value": "start.js",
```
Comment (Romitou) [inline nodejs/nodejs.json:32]: This file is usually called `index.js`.
```suggestion
      "value": "index.js",
```

File: nodejs/nodejs.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      "type": "string",
+      "required": true,
+      "desc": "Which file will node js start running",
```
Comment (Romitou) [inline nodejs/nodejs.json:35]:
```suggestion
      "desc": "Which file NodeJS will run",
```

File: nodejs/nodejs.json (hunk 2 above)
Comment (zigazajc007) [reply nodejs/nodejs.json:5]: Sorry, I just meant to "submit" FiveM. I still didn't finish NodeJS.
Also, I have no clues about signals.

**Pull request closed without merging**